database/schema.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE musicAlbum (
+  id INT PRIMARY KEY REFERENCES item(id),
+  on_spotify BOOLEAN NOT NULL
+);
```

Comment:
```suggestion
CREATE TABLE music_album (
  id INT PRIMARY KEY REFERENCES item(id),
  on_spotify BOOLEAN NOT NULL,
  FOREIGN KEY (id) REFERENCES item (id) ON DELETE RESTRICT ON UPDATE CASCADE
);
```

Reply:
- [x] I think it will be better if you add a foreign key to make a relationship between music_album and item tables.